### PR TITLE
docs: Minor fixes on the docs

### PR DESCRIPTION
**Pull request by waynesun09**
Fix minor problems after went through the doc.

Signed-off-by: Wayne Sun <gsun@redhat.com>

### CODE REVIEW

File: docs/source/config_layout.rst [hunk 1]
```diff
@@ -28,7 +28,7 @@ value is optional and defaults to 1 when it's not present.
 The `host_groups` field contains an array of Ansible inventory groups into which all
 the hosts under this hash will get placed. This value is optional and will default to
 an empty list when not filled. In that case, the host will be named into the inventory
-with its host vars, but it will not 
+with its host vars, and add to default 'all' group.
```
Comment (herlo) [inline docs/source/config_layout.rst]: I believe this should be "... and added to the default ..."

Yes?

Comment (waynesun09) [inline docs/source/config_layout.rst]: Sure, updated

**Comment by waynesun09** [comment]
@herlo I didn't touch the CLI `linchpincli_linchpin_*` part, only one format problem I found though, are CLI part auto generated? If so, which extension do you use for generate it?

**Comment by herlo** [comment]
Please merge at your earliest convenience. :+1:

**Pull request merged**